subsys/net/openthread/Kconfig

```diff
@@ -36,7 +36,9 @@ config OPENTHREAD_NRF_SECURITY_PSA
 	select MBEDTLS_ENABLE_HEAP
 	select MBEDTLS_TLS_LIBRARY
 	select MBEDTLS_PSA_CRYPTO_STORAGE_C if !BUILD_WITH_TFM
-	select PSA_NATIVE_ITS if !BUILD_WITH_TFM
+	select SECURE_STORAGE if !BUILD_WITH_TFM
+	select SETTINGS if !BUILD_WITH_TFM
```

Comment:
`SECURE_STORAGE_STORAGE_SETTINGS_BACKEND` is default selected

Reply:
If your default changes, then this gets invalid. It's not about if it is correct now, but how it will act when changes are made. Relying on defaults in different parts of the system creates something that is hard to maintain.

If SECURE_STORAGE_STORAGE_SETTINGS_BACKEND selected SETTINGS it would be different.

Now it is selecting an implicit dependency, and it is hard to see why.

Reply:
Can't select a choice, so disregard this comment.

Reply:
Grouped the dependencies into an own entry